### ISSUE TITLE
Tidy data and support for Visualization and other output presentation

### ISSUE THREAD

**Issue by EliLazarus** [julia-mpsge/MPSGE.jl#139]

**Comment by mitchphillipson** [comment]
Yes. Best solution might be to have a method to convert indexed sectors to DataFrames. I have this on a global scale with `generate_report`, but a micro scale would be nice too. Column for variable name, index columns, and value. 

So if `Y` is a variable with domain `A, B` we get a DataFrame with four columns, `:variable_name`, `:A`, `:B`, `:value`. 

In fact we could be clever here and set the axis names on creation of an indexed variable. And IndexedVariable is just a DenseAxisArray which space for axis names. I was prototyping something earlier and will circle back to this.

**Comment by EliLazarus** [comment]
I don't know how far we want to take this as an issue, but there's probably a list of intersting, useful and slightly complicated output and report methods that might be nice to offer. 
Probably for some simple ones like a bar chart or scatter plot of results, better to just show in tutorial.

**Comment by mitchphillipson** [comment]
I'm moving this to #93 as a documentation fix.
